x/superfluid/types/msgs.go

```diff
@@ -392,3 +393,40 @@ func (msg MsgAddToConcentratedLiquiditySuperfluidPosition) GetSigners() []sdk.Ac
 	}
 	return []sdk.AccAddress{sender}
 }
+
```

Comment:
is adding CLI going to be in a different PR?

Reply:
added CLI here https://github.com/osmosis-labs/osmosis/pull/5745/commits/ccd45be0f984850c7ddaa11d5b62dd0b93542c04